website/artisan.msbuildproj

```diff
@@ -1,6 +1,7 @@
-<Project Sdk="Peachpie.NET.Sdk/0.9.38">
+<Project Sdk="Peachpie.NET.Sdk/0.9.9-dev">
```

Comment:
all `0.9.9-dev` versions should be changed to `0.9.45` (or the latest) on production